Tests/AmplitudeTests/Supports/TestUtilities.swift

```diff
@@ -129,6 +129,10 @@ class FakeHttpClient: HttpClient {
         completion(Result.success(200))
         return nil
     }
+    override func getDate() -> Date {
```

Comment:
nit: missing newline
```suggestion

    override func getDate() -> Date {
```